test/functional/DDR_Test/src/j9vm/test/ddrext/Constants.java

```diff
@@ -291,7 +291,7 @@
 
 	public static final String J9CLASSSHAPE_CMD = "j9classshape";
 	public static final String J9CLASSSHAPE_TEST_CLASS = "j9vm/test/corehelper/SimpleThread";
```

Comment:
This should change to `"j9vm/test/corehelper/SimpleThread$DumperThread"` otherwise the test might actually find `SimpleThread` instead of `SimpleThread.DumperThread` and subsequently fail.